src/System.Management.Automation/resources/TabCompletionStrings.resx

```diff
@@ -346,7 +346,7 @@ A script block used to calculate the value of the new property.</value>
 Define how the values are displayed in a column.
 Valid values are 'left', 'center', or 'right'.</value>
   </data>
-  <data name="FormatstringHashtableKeyDescription" xml:space="preserve">
+  <data name="FormatStringHashtableKeyDescription" xml:space="preserve">
```

Comment:
I can not find where it is used.

Reply:
Some of the format commands use it, like: `ls | Format-Table @{<Tab>`